R/run.R

```diff
@@ -723,11 +728,34 @@ CmdStanProcs <- R6::R6Class(
         }
       }
     },
+    print_progress = function(){
+      # get the width of the widest id (sets # leading zeros on all)
+      id_width = nchar(as.character(length(self$proc_ids())))
+      # flush the console at outset just in case it hasn't been updated in a while
+      flush.console()
+      # loop over processes
+      for(id in self$proc_ids()){
+        cat( format(id, width=id_width), ': ', sep='' )
+        # set %-compute text
+        if(private$iter_done_[[id]]==0){
+          # waiting-to-start text is ...
+          cat(' ... | ') # same # of chars as '100% '
+        }else{
+          # if any done, show %
+          cat( format(floor(100*private$iter_done_[[id]]/private$iter_),width=3), '% | ', sep='' )
+        }
+      }
```

Comment:
```suggestion
      for (id in self$proc_ids()) {
        cat(format(id, width = id_width), ': ', sep = '')
        # set %-compute text
        if (private$iter_done_[[id]] == 0) {
          # waiting-to-start text is ...
          cat(' ... | ') # same # of chars as '100% '
        } else {
          # if any done, show %
          cat(format(floor(100*private$iter_done_[[id]]/private$iter_), width = 3), '% | ', sep = '' )
        }
      }
```

Just spacing changes to follow coding style